src/main/java/au/gov/nla/pickslip/service/FolioService.java

```
@@ -248,4 +306,118 @@ protected JsonNode folioApiGetRequests() throws IOException {
+            folioRequestJson.at("/requesterId")
+                .asText(null),
+            folioRequestJson.at("/status")
+                .asText(null),
+            folioRequestJson.at("/cancellationAdditionalInformation")
+                .asText(null),
+            folioRequestJson.at("/position")
+                .asText(null),
+            new FolioRequest.Instance(folioRequestJson.at("/instance/title")
+                .asText(null)),
+            new FolioRequest.Item(
+                folioRequestJson.at("/item/barcode")
+                    .asText(null),
+                folioRequestJson.at("/item/callNumber")
+                    .asText(null),
+                new FolioRequest.Item.Location(
+                    folioRequestJson.at("/item/location/name")
+                        .asText(null),
+                    folioRequestJson.at("/item/location/code")
+                        .asText(null))),
+            new FolioRequest.Requester(
+                folioRequestJson.at("/requester/barcode")
+                    .asText(null),
+                folioRequestJson.at("/requester/patronGroupGroup")
+                    .asText(null)),
+            toList(folioRequestJson.at("/tagList")
+                .elements()));
+  }
+
+  public void updateRequest(final RequestNoteDto requestNoteDto) throws IOException {
+    FOLIORequestsRetrieverAPI folioRequestsRetrieverAPI =
+        new FOLIORequestsRetrieverAPI(folioOkapiCredentials);
+    JsonNode folioRequestJson =
+        folioRequestsRetrieverAPI.getRequestById(requestNoteDto.getRequestId());
+    if (folioRequestJson != null) {
+      ((ObjectNode) folioRequestJson).put("cancellationAdditionalInformation",
+          requestNoteDto.getCancellationAdditionalInformation());
+      folioRequestsRetrieverAPI.updateRequest(requestNoteDto.getRequestId(), folioRequestJson);
+    }
+    else {
+      log.error("Could not retrieve request with id: {}", requestNoteDto.getRequestId());
```

Should this result in feedback to the user?  (Imagining it'll fail silently apart from log file message.)